tests/testsuite/cargo_remove/invalid_section/stderr.log

```diff
@@ -1,2 +1,2 @@
     Removing docopt from build-dependencies
-error: the dependency `docopt` could not be found in `build-dependencies`.
+error: the dependency `docopt` could not be found in `build-dependencies`. But it was found in `dependencies`.
```

Comment:
Can you update it so that each commit that will be merged passes tests?

Reply:
I've reabsed them into one commit.